.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Dodałem plik z regułami formatowania kodu, i proponuję zmienić `tabWidth` na 4 bo i więcej jest różnych elementów tym płytsze są te wcięcia i w vs code słabo to widać i przez to jest mniej czytelne (mam takie pionowe kreski łączące tagi)

Np. tab 2

![image](https://github.com/pawelNu/compstore-ui/assets/93542936/6fdb6925-b153-41c1-ad17-87915385e403)

tab 4

![image](https://github.com/pawelNu/compstore-ui/assets/93542936/985fecb6-00b4-4043-a25b-8e5ba70c3bcc)

Nie wiem na ile używasz tego, ale mi to bardzo pomaga orientować się gdzie się tag zaczyna, a gdzie kończy.

Reply:
Okej, jestem za

Reply:
Oki, to dodam kolejnego commita z formatowaniem całego projektu z tab 4.